evaluator/src/test/kotlin/PackageRuleTest.kt

```diff
@@ -0,0 +1,127 @@
+
+    init {
+        "hasLicense()" should {
+            "return true if the package has concluded licenses" {
+                val rule = PackageRule(ruleSet, "test", packageWithOnlyConcludedLicense, emptyList())
+                val matcher = rule.hasLicense()
+
+                matcher.matches() shouldBe true
+            }
+
+            "return true if the package has declared licenses" {
+                val rule = PackageRule(ruleSet, "test", packageWithOnlyDeclaredLicense, emptyList())
+                val matcher = rule.hasLicense()
+
+                matcher.matches() shouldBe true
+            }
+
+            "return true if the package has detected licenses" {
+                val rule = PackageRule(ruleSet, "test", packageWithoutLicense, licenseFindings)
+                val matcher = rule.hasLicense()
+
+                matcher.matches() shouldBe true
+            }
+
+            "return false if the package has no license" {
+                val rule = PackageRule(ruleSet, "test", packageWithoutLicense, emptyList())
+                val matcher = rule.hasLicense()
+
+                matcher.matches() shouldBe false
+            }
+        }
+
+        "isExcluded()" should {
+            "return true if the package is excluded" {
+                val rule = PackageRule(ruleSet, "test", packageExcluded, emptyList())
+                val matcher = rule.isExcluded()
+
+                matcher.matches() shouldBe true
+            }
+
+            "return false if the package is not excluded" {
+                val rule = PackageRule(ruleSet, "test", packageWithoutLicense, emptyList())
+                val matcher = rule.isExcluded()
+
+                matcher.matches() shouldBe false
+            }
+        }
+
+        "isFromOrg()" should {
+            "return true if the package is from org" {
+                val rule = PackageRule(ruleSet, "test", packageWithoutLicense, emptyList())
+                val matcher = rule.isFromOrg("here")
+
+                matcher.matches() shouldBe true
+            }
+
+            "return false if the package is not from org" {
+                val rule = PackageRule(ruleSet, "test", packageWithoutLicense, emptyList())
+                val matcher = rule.isFromOrg("unknown")
+
+                matcher.matches() shouldBe false
+            }
+        }
+
+        "isType()" should {
+            "return true if the package has the provided type" {
+                val rule = PackageRule(ruleSet, "test", packageWithoutLicense, emptyList())
+                val matcher = rule.isType("Maven")
+
+                matcher.matches() shouldBe true
+            }
+
+            "return false if the package has not the provided type" {
+                val rule = PackageRule(ruleSet, "test", packageWithoutLicense, emptyList())
+                val matcher = rule.isType("Gradle")
+
+                matcher.matches() shouldBe false
+            }
+        }
+
+        "isSpdxLicense()" should {
```

Comment:
can you add a test which illustrates the behavior in case of multiple declared licenses ?
e.g. a spdx one and a non spdx one?